### PR TITLE
config/gradle_8.6

### PR DESCRIPTION
**Pull request by Hounshell**
Patch up demo build files and use a better technique for detecting whether the projects are being run as part of the AppLovin workspace or as a standalone demo.

### CODE REVIEW

File: AppLovin MAX Demo App - Kotlin/app/build.gradle
Comment (santoshbagadi) [inline AppLovin MAX Demo App - Kotlin/app/build.gradle:1]: We should update the compile and target SDK versions in this file to 34 too.

**Review by santoshbagadi — APPROVED** [review]

**Review by thomasmso — APPROVED** [review]

File: AppLovin MAX Demo App - Java/app/build.gradle [hunk 1]
```diff
@@ -36,7 +38,7 @@ android {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.aar'])
```
Comment (Lorenzo45) [inline AppLovin MAX Demo App - Java/app/build.gradle:41]: What exactly does this change do? Should we add it to the Kotlin demo app as well?

**Review by Lorenzo45 — APPROVED** [review]

File: AppLovin MAX Demo App - Java/app/build.gradle (hunk 1 above)
Comment (Hounshell) [reply AppLovin MAX Demo App - Java/app/build.gradle:41]: It doesn't really do anything, but it felt like a good practice. The old one basically says "treat any jars in the libs folder as a dependency". The change expands it include aars. We don't have a libs folder, so it does nothing, but it is handy to be able to drop an aar in there and have it just work.

File: AppLovin MAX Demo App - Kotlin/app/build.gradle
Comment (Hounshell) [reply AppLovin MAX Demo App - Kotlin/app/build.gradle:1]: Done.

**Pull request merged**